hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/procedure/TestClusteringProcedure.scala

```diff
@@ -325,7 +335,7 @@ class TestClusteringProcedure extends HoodieSparkSqlTestBase {
           assertResult(3)(clusteringPlan.get().getInputGroups.size())
 
           // No pending clustering instant
-          checkAnswer(s"call show_clustering(table => '$tableName')")()
+          spark.sql(s"call show_clustering(table => '$tableName')").show()
```

Comment:
why change `checkAnswer` to `spark.sql`?

Reply:
I will change this test case to check more return result info